src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1091,13 +1109,24 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(string taskId, TesT
 
             if (isPublic.DockerInDockerImage)
             {
-                sb.AppendLinuxLine($"(grep -q alpine /etc/os-release && apk add bash || :) && \\");  // Install bash if running on alpine (will be the case if running inside "docker" image)
+                sb.AppendLinuxLine($"(grep -q alpine /etc/os-release && apk add bash wget || :) && \\");  // Install bash if running on alpine (will be the case if running inside "docker" image)
```

Comment:
You probably will also want to add `wget` to [`microsoft/CromwellOnAzure/src/deploy-cromwell-on-azure/samples/docker-dockerfile`](https://github.com/microsoft/CromwellOnAzure/blob/main/src/deploy-cromwell-on-azure/samples/docker-dockerfile)